src/Microsoft.AspNet.Server.Kestrel/Infrastructure/TaskUtilities.cs

```diff
@@ -7,13 +7,10 @@ namespace Microsoft.AspNet.Server.Kestrel.Infrastructure
 {
     public static class TaskUtilities
     {
-        public static Task CompletedTask = NewCompletedTask();
-
-        private static Task NewCompletedTask()
-        {
-            var tcs = new TaskCompletionSource<int>();
-            tcs.SetResult(0);
-            return tcs.Task;
-        }
+#if DNXCORE50
```

Comment:
Changed, but failed on Travis post tests time out; will keep spinning the wheel till it passes.